packages/lab/src/components/Flow/stories/ToyStory.tsx

```diff
@@ -0,0 +1,32 @@
+import { theme } from "@hitachivantara/uikit-styles";
+import { HvFlowNode } from "../Node/Node";
+
+export const ToyStory = (props) => {
+  return (
+    <HvFlowNode
```

Comment:
I think we want "Toy Story" and "Toy Story Description" for the `title` and `description`, respectively. 
Also, `color` is no longer needed (same for the other nodes).